docs/ZendeskMultiObjects-batchsource.md

```diff
@@ -44,7 +46,7 @@ or equal to the specified date. The date must be provided in the date format:
 
 Start Date is required for batch objects like: Ticket Comments, Organizations, Ticket Metric Events, Tickets, Users.
 
-**End Date:** Filter data to include only records which have Zendesk modified date is less than 
+**End Date:** Filter data to include only records that have Zendesk modified date is less than 
```

Comment:
Missing some words:
"Filter data to include only records that have a Zendesk modified date that is less than...."
See: https://docs.google.com/document/d/1T8eCPI9rgZm7tL9OfhWb2EtUGfdhMYY7aryfos7deKM/edit?resourcekey=0-JCNeuQYONXE8cB9WS3iEJw#

Reply:
added